src/main/resources/anniv5Resources/localization/eng/farmerpack/Cardstrings.json

```diff
@@ -1,8 +1,13 @@
 {
   "${ModID}:Fertilizer": {
     "NAME": "Fertilizer",
-    "DESCRIPTION": "Draw a card. NL Exhaust.",
-    "UPGRADE_DESCRIPTION": "Draw !M! cards. NL Exhaust."
+    "DESCRIPTION": "Draw a card. NL Exhaust. NL This is a Status",
```

Comment:
Is the status text really needed? that's what the type is for.
Also, would need a period